src/DependencyInjection/ConnecthollandUserExtension.php

```diff
@@ -27,6 +27,10 @@ class ConnecthollandUserExtension extends Extension implements ExtensionInterfac
 {
     public function load(array $configs, ContainerBuilder $container)
     {
+        $configuration = new Configuration();
+        $config        = $this->processConfiguration($configuration, $configs);
+        $container->setParameter('connectholland_user.user_class', $config['user_class']);
```

Comment:
This should not be needed, you should 'fix' the service definition of the service that needs it. IIRC replaceArgument call on it's definition